_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Contributing a Blog Post to pulpproject.org
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `YYYY-MM-DD-title-of-post.md` e.g.
+   `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+1. Commit your changes to the `gh-pages` branch.
+
+1. Push your `gh-pages` branch to GitHub.
+
+1. Preview the blog at `https://<username>.github.io/pulpproject.org/blog/`. You can also build the
```

Comment:
More explicit wording pointing this out as an alternative...

s/You can also build the site locally and view your blog post/Another option to preview your blog post (without Github) is to build it locally/.

I think the `build it locally` part can be the link to the README.md and the last sentence can be removed.